src/middlewared/middlewared/etc_files/local/nut/upsd.conf

```diff
@@ -2,10 +2,10 @@
     ups_config = middleware.call_sync('ups.config')
 %>\
 % if ups_config['rmonitor']:
-LISTEN 0.0.0.0
-LISTEN ::0
+LISTEN 0.0.0.0 ${ups_config['remoteport']}
```

Comment:
You should add some validation on `remoteport` since it's now it's in use for both modes.